Documentation/dl_build.md

```diff
@@ -48,14 +48,18 @@ $ $GOPATH/bin/etcd
 
 ### Setting a key
 
-Run the following:
+If etcd is built without using GOPATH, run the following:
```

Comment:
i do not really understand this. how to run/build etcd has nothing to do with GOPATH.

Reply:
I think this is referring to the previous section, where two methods were provided to build the latest version of etcd:
1. To build etcd from the master branch **without a GOPATH** using the official build script
2. To build a vendored etcd from the master branch via go get, where it says **GOPATH should be set**